custom/icds_reports/utils/aggregation_helpers/ccs_record_monthly.py

```diff
@@ -52,62 +52,139 @@ def tablename(self):
     def drop_table_query(self):
         return 'DELETE FROM "{}"'.format(self.tablename)
 
+    @property
+    def person_case_ucr_tablename(self):
+        doc_id = StaticDataSourceConfiguration.get_doc_id(self.domain, 'static-person_cases_v2')
+        config, _ = get_datasource_config(doc_id, self.domain)
+        return get_table_name(self.domain, config.table_id).decode('utf-8')
+
     def aggregation_query(self):
+        start_month_string = self.month.strftime("'%Y-%m-%d'::date")
+        end_month_string = (self.month + relativedelta(months=1) - relativedelta(days=1)).strftime("'%Y-%m-%d'::date")
+        age_in_days = "({} - case_list.dob)::integer".format(end_month_string)
+        age_in_months_end = "({} / 30.4 )".format(age_in_days)
+        open_in_month = ("({} - case_list.opened_on::date)::integer >= 0"
+                         " AND (case_list.closed = 0"
+                         " OR (case_list.closed_on::date - {})::integer > 0)").format(end_month_string,
+                                                                                       start_month_string)
+        alive_in_month = "(case_list.date_death is null OR case_list.date_death-{}>0)".format(
+            start_month_string)
+        seeking_services = "(case_list.is_availing = 1 AND case_list.is_migrated = 0)"
```

Comment:
does it not need to be open to be seeking services? does the old version allow this?

Reply:
yeah I think So if I consider the definition of seeking in child_health and ccs_Record is same